### ISSUE TITLE
Add support for binding to mouse buttons

### ISSUE THREAD

**Issue by robere2** [bugfroggy/Quickplay2.0#140]
Mouse buttons 3 and up should support binding to Quickplay keybinds. Mouse button 1 and 2 shouldn't be supported since those are the main buttons used to interact with the Quickplay GUIs

**Comment by robere2** [comment]
Note, Mouse buttons 4 and up require Patcher to support use in Minecraft.

**Comment by yodaluca23** [comment]
Yes!! Please add this I was SOO disappointed when I got this mod and realized that wouldn't work!
